stable/prometheus-operator/templates/prometheus/rules/general.rules.yaml

```diff
@@ -25,9 +27,19 @@ spec:
       for: 10m
       labels:
         severity: warning
-    - alert: DeadMansSwitch
+    - alert: Watchdog
```

Comment:
I would consider this a major change. 
- People may have (and we do!) custom configurations around this alert that will all need updating.
- https://github.com/helm/charts/blob/master/stable/prometheus-operator/values.yaml#L108 will need to get changed too

Reply:
Should be fixed now, see latest commit.

Reply:
> * People may have (and we do!) custom configurations around this alert that will all need updating. Anything using the old name will break.

@vsliouniaev Yes, it breaks my [workaround](https://stackoverflow.com/a/54814033/7641078).

Thankfully I read your warning before upgrading :+1: